lib/command.js

```diff
@@ -1084,6 +1084,26 @@ Expecting one of '${allowedValues.join("', '")}'`);
     return hookResult;
   }
 
+  /**
+   * Invoke help directly if possible, or dispatch if necessary.
+   * e.g. help foo
+   *
+   * @api private
+   */
+
+  _dispatchHelpCommand(subcommandName) {
+    if (!subcommandName) {
+      this.help();
+    }
+    const subCommand = this._findCommand(subcommandName);
```

Comment:
If `subcommandName` is `undefined`, `this._findCommand(subcommandName)` also returns `undefined`.

```js
if (!subcommandName) {
  this.help();
} else {
  const subCommand = this._findCommand(subcommandName);
  ...
}
```

Reply:
The help method calls exit, so the following code will not be reached if `subcommandName` is `undefined`.

Reply:
That makes sense.